src/components/Header/styles.ts

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components/native';
+import {Color} from '../../ui/colors';
```

Comment:
lets rename folder `ui` to `theme`, and add alias for that folder. it should be like that:
`import {Color} from '@theme/colors';`